lib/data_hygiene/publishing_api_sync_check.rb

```diff
@@ -86,6 +90,26 @@ def perform(output: true)
 
   private
 
+    def add_translations_expectation
+      add_expectation("translations") do |_, record|
+        if translation_locales_for(record).empty?
```

Comment:
This isn't needed, if `translation_locales_for(record)` is empty, the `translations_present_and_have_correct_basepath?` will return true anyway.